public/javascripts/continuousPlay.js

```diff
@@ -19,28 +20,21 @@ if (pageNum === 5) {
 
 //on clicking next
 function nextGuess() {
-  const guesses = document.querySelectorAll('input[name="guess"]');
+  const userGuess = document.querySelector('input[name="guess"]:checked');
   const gameName = document.getElementById("game-name").innerText;
   const imgLink = document.getElementById("img-link").innerText;
 
   //saving currect guess data
-  userGuessData[`gameName${pageNum}`] = gameName;
-  userGuessData[`imgLink${pageNum}`] = imgLink;
+  currentUserGuessData["gameName"] = gameName;
+  currentUserGuessData["imgLink"] = imgLink;
 
   //checking for user guess and saving guess status (correct/wrong)
-  let thereIsCheckedGuess = 0;
-  for (const guess of guesses) {
-    if (guess.checked) {
-      thereIsCheckedGuess++;
-      guess.value === gameName
-        ? (userGuessData[`userGuess${pageNum}`] = "correct")
-        : (userGuessData[`userGuess${pageNum}`] = "wrong");
-    }
-  }
-  if (!thereIsCheckedGuess) {
-    userGuessData[`userGuess${pageNum}`] = "wrong";
+  if (userGuess) {
```

Comment:
I'm not sure, but is this condition still needed?

Reply:
yep, had it removed initially but ran into some issues, so added it back in

Reply:
Right, am still a bit unfamiliar with this thing, so forgot about it. Thanks!